com.unity.render-pipelines.high-definition/Documentation~/Density-Volume.md

```diff
@@ -22,27 +22,27 @@ To create a Density Volume, right click in the Hierarchy and select __Rendering
 | **Invert Blend**             | Reverses the direction of the fade. Setting the Blend Distances on each axis to its maximum possible value preserves the fog at the center of the Volume and fades the edges. Inverting the blend fades the center and preserves the edges instead. |
 | **Distance Fade Start**             | Distance from the camera at which the Density Volume starts to fade out. This is useful when optimizing a scene with many Density Volumes and making the more distant ones disappear|
 | **Distance Fade End**             | Distance from the camera at which the Density Volume has completely fade out. This is useful when optimizing a scene with many Density Volumes and making the more distant ones disappear|
-| **Density Mask Texture**     | Specifies a 3D texture mapped to the interior of the Volume. The Density Volume only uses the alpha channel of the texture. The value of the texture acts as a density multiplier. A value of 0 in the Texture results in a Volume of 0 density, and the texture value of 1 results in the original constant (homogeneous) volume. |
+| **Density Mask Texture**     | Specifies a 3D texture mapped to the interior of the Volume. The Density Volume only uses the RGB channels of the texture for the fog color and A for the fog density multiplier. A value of 0 in the Texture alpha channel results in a Volume of 0 density, and the value of 1 results in the original constant (homogeneous) volume. |
 | **Scroll Speed**             | Specifies the speed (per-axis) at which the Density Volume scrolls the texture. If you set every axis to 0, the Density Volume does not scroll the texture and the fog is static. |
 | **Tiling**                   | Specifies the per-axis tiling rate of the texture. For example, setting the x-axis component to 2 means that the texture repeats 2 times on the x-axis within the interior of the volume. |
 
+## Related Settings in the HDRP Asset
 
+There are two settings related to density volumes you can find in the HDRP Asset settings: **Max Density Volume Size** and **Max Density Volume On Screen** and they can be found under the Lighting category.
 
-## Creating a Density Mask Texture
-
-1. In image-editing software of your choice, prepare a grayscale flipbook texture and [import it as a 3D texture](https://docs.unity3d.com/2020.2/Documentation/Manual/class-Texture3D.html). For example, a texture of size 1024x32 describes a 3D texture of size 32x32x32 with 32 slices laid out one after another.
+![](Images/HDRPAsset_VolumetricSettings.png)
 
-2. Additionnaly, in the Texture Import Settings:
-    1. Set the __Texture Type__ to __Single Channel__.
+The **Max Density Volume Size** controls the maximum resolution you can use for the **Density Mask Texture**, changing this setting can have a big impact on memory. Please refer to the info box below to see how much memory will be allocated by the Density Volume system with your current settings.
```

Comment:
The **Max Density Volume Size** controls the maximum resolution you can use for the **Density Mask Texture**. Changing this setting can have a large impact on memory usage. For information on how much memory HDRP allocates for the Density Volume system, see the info box below this setting in the HDRP Asset.